discovery/bootstrapper.go

```diff
@@ -247,13 +249,14 @@ var _ NetworkPeerBootstrapper = (*ChannelGraphBootstrapper)(nil)
 // NewDNSSeedBootstrapper returns a new instance of the DNSSeedBootstrapper.
 // The set of passed seeds should point to DNS servers that properly implement
 // Lighting's DNS peer bootstrapping protocol as defined in BOLT-0010.
-//
-//
-// TODO(roasbeef): add a lookUpFunc param to pass in, so can divert queries
-// over Tor in future
-func NewDNSSeedBootstrapper(seeds []string) (NetworkPeerBootstrapper, error) {
+func NewDNSSeedBootstrapper(seeds []string, lookupHost func(string) ([]string, error),
```

Comment:
The `godoc` comment above should be updated to document the expected arguments and behavior of the two new closure functions passed in.